### DISCUSSION TITLE
_layouts images in InlineImage

### DISCUSSION THREAD

**Discussion by Nevinia210** [pnp/pnpcore#695] [Q&A] [answered]
Hello everyone,

Is it possible to have inline image from system images("/_layouts/images/icpdf.png")?
I'm getting following error while trying to do that:
The file /sites/mytestsite/_layouts/images/icpdf.png does not exist.

when i try to access that file with url, its there

**Comment by jansenbe** [comment, marked as answer]
@Nevinia210 , this is currently not supported as the code tries to lookup the image, which fails for the "old" images hosted under _layouts. Can you create a issue for this to ensure the ask is not getting lost. Thx.
